.ci/generate_package.py

```diff
@@ -145,7 +146,7 @@ def main():
                                  'BEAR_NAME': bear_name,
                                  'ENTRY': 'coala' + bear_name}
 
-            create_file_from_template(os.path.join('bears', 'setup.py.in'),
+            create_file_from_template(os.path.join(current_dir, 'setup.py.in'),
```

Comment:
create an argparse arg for the file filename of the template ; the tests already allow this extra arg to be added once in the setUp .

It is very important that we use explicit paths so it isnt 'magic' - they need to read this file, and hopefully improve it, before uploading crappy packages to pypi with that template.

my preference would be to keep this file in .moban.dt , but I have no objection if you feel it is better in `.ci`